testing/libtorrent-rasterbar/APKBUILD

```diff
@@ -1,40 +1,43 @@
 # Contributor: August Klein <amatcoder@gmail.com>
 # Maintainer: August Klein <amatcoder@gmail.com>
 pkgname=libtorrent-rasterbar
-pkgver=1.1.12
-pkgrel=1
+pkgver=1.1.13
+pkgrel=0
 pkgdesc="Feature complete C++ bittorrent implementation"
 url="http://www.rasterbar.com/products/libtorrent"
 arch="all"
 license="BSD-3-Clause"
-depends=""
 depends_dev="boost-dev openssl-dev python2-dev"
 makedepends="$depends_dev linux-headers"
-subpackages="$pkgname-dev"
-source="https://github.com/arvidn/libtorrent/releases/download/libtorrent_${pkgver//./_}/$pkgname-$pkgver.tar.gz"
-
-builddir="$srcdir"/$pkgname-$pkgver
+subpackages="py2-$pkgname:_py2 $pkgname-static $pkgname-dev"
```

Comment:
why adding python2 subpackage?
Py2 is about to get dropped, no need to add packages for that.